video_creation/final_video.py

```diff
@@ -287,8 +285,12 @@ def make_final_video(
     subreddit = settings.config["reddit"]["thread"]["subreddit"]
 
     if not exists(f"./results/{subreddit}"):
-        print_substep("The results folder didn't exist so I made it")
+        print_substep("The 'results' folder didn't exist so I made it")
         os.makedirs(f"./results/{subreddit}")
+    
+    if not exists(f"./results/{subreddit}/OnlyTTS") and allowOnlyTTSFolder:
+        print_substep("The 'OnlyTTS' folder didn't exist so I made it")
```

Comment:
Try not to use a first-person perspective. or any person.

do something like         print_substep("The 'OnlyTTS' folder could not be found so it was automatically created.")

Reply:
I just kept the code as the same as `the master and develop branch`, i agree with you that isn't good using first person, but i am just keeping the repo syntax:
![image](https://user-images.githubusercontent.com/105776775/234146969-995cc82a-7787-4f66-a6c7-9cbdf89916e5.png)

Reply:
So i'll change two of this, like: "The 'results' folder could not be found so it was automatically created.
Will be better i agree with you, but my thought was keep the code syntax